scripts/updateTransferGasLimit.ts

```diff
@@ -0,0 +1,80 @@
+import { ethers } from 'hardhat';
+import { getChainConfig } from './chainConfig';
+import { logger } from '../deployUtils';
+
+async function main() {
+  // Use the first configured signer on the selected network
+  const signers = await ethers.getSigners();
+  const signer = signers[0];
```

Comment:
how is this populated ?
how can we make sure the key used of `PRIVATE_KEY_FOR_BATCHER_CONTRACT_DEPLOYMENT` ?